### ISSUE TITLE
menu color is hard-coded

### ISSUE THREAD

**Issue by windowsrefund** [duvholt/i3-pager#35]
Greetings. I'm just wondering where colors are determined as I have a menu I've configured which is being rendered with strange colors.

I don't normally run a `bar` but I enabled one just to make sure it my menu was behaving. This image shows the menu correctly colored:

![i3-menu-good](https://github.com/user-attachments/assets/c34975ae-3e32-4233-a123-dcef2566ec4e)

The same menu however, displayed via this `i3-pager` widget is rendered using totally different colors:

![i3-menu-bad](https://github.com/user-attachments/assets/8adbf547-b182-425d-970f-8cb3f2d88c7a)

Just so it's clear, I'm referring to an i3 menu

```
set $mode_menu MENUS: (b)ase16 (i)3
bindsym $mod+i mode "$mode_menu"

```

Thanks in advance for any info.

**Comment by windowsrefund** [comment]
Looks like this is [hard-coded](https://github.com/duvholt/i3-pager/blob/a30babac07b828a4809d9a41c5ce7ffc616f2878/plasmoid/contents/ui/main.qml#L201C29-L201C36)

**Comment by duvholt** [comment]
Yep, the colors for the binding mode are hardcoded. Originally all colors were hardcoded, but some were changed to follow the plasma theme. The original usecase was just a way for me to see if I'm in resize mode.

I think it might make sense to allow all colors to be user configurable and we can probably take some inspiration from i3bar's colors: https://i3wm.org/docs/userguide.html#_colors
